app/services/payout/bitflyer_job_implementation.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Payout
+  class BitflyerJobImplementation
```

Comment:
Nit picking, but this is a service rather than a job.

The underlying issue I see that existed before this code is that the caller could be killed for whatever reason before it completes without ever retrying. As a a result I'm punting it to https://github.com/brave-intl/publishers/issues/3245